src/libs/events.js

```diff
@@ -8,6 +8,8 @@ const eventsList = {
   applicationLaunch: 'application:launch',
   applicationCreate: 'application:create',
   applicationDelete: 'application:delete',
+  applicationPause: 'application:pause',
```

Comment:
Thanks for remembering to add Mixpanel Events! Checked in the mixpanel dashboard and these events look good!